spec/models/claims/totals_spec.rb

```diff
@@ -127,27 +129,27 @@
           claim.fees << create(:misc_fee, rate: 10.5, quantity: 3)
           claim.fees << create(:misc_fee, rate: 6.25, quantity: 1)
           claim.save!
-          expect_totals_to_be(claim, 0.0, 0.0, 37.75, 6.61, 0.0, 0.0)  # VAT £6.61 calculated using the test VAT rate of 17.5%
+          expect_totals_to_be(claim, 0.0, 0.0, 37.75, 7.55, 0.0, 0.0) # VAT £6.61 calculated using the test VAT rate of 20.0%
```

Comment:
💭  I know this is not your initial change, but the first feeling I get looking at this expectation is that I don't have a clue what is being asserted. Having looked at the actual implementation of the  expectation, I would argue that `DRY` in this case its not necessary good or clear.

I would suggest, at least, changing the interface to be something more expressive, for example (not necessary this, but you get the idea):

```ruby
expect(claim).to have_totals(
  expenses_total: 9.99,
  expenses_vat:  2.0,
  fees_total: 37.75,
  fees_vat: 7.55,
  disbursements_total: 0.0,
  disbursements_vat: 0.0,
  total: 9.99 + 37.75 + 0.0,
  vat_amount: 2.0 + 7.55 + 0.0
)
```

Reply:
indeed 😢 , i changed to get it passing (should have been a separate commit to get tests passing) and have done some spec refactoring generally. It would be good to implement as you  suggest as above, but started to get refactoring creep

Reply:
...and ideally it should test the individual values as separate tests